charts/gsp-cluster/templates/02-gsp-system/prometheus-rules.yaml

```
@@ -9,7 +9,7 @@ spec:
     - alert: TargetDown
       annotations:
         message: '{{`{{ $value }}`}}% of the {{`{{ $labels.job }}`}} targets are down.'
-      expr: 100 * (count(up == 0) BY (job) / count(up) BY (job)) > 10
+      expr: 100 * (count(up == 0) WITHOUT (instance, pod) / count(up) WITHOUT (instance, pod)) > 10
```

```suggestion
      expr: 100 * (avg without(instance, pod) (up)) < 90
```
⛳️ 

aggregating `up` with `avg` gives a sense of "what proportion of my service is up".  It means the same thing but is shorter and perhaps slightly more idiomatic